app/services/versioned_files_service.rb

```diff
@@ -21,12 +21,9 @@ def self.versioned_files?(druid:)
   def initialize(druid:)
     @druid = druid
     @paths = Paths.new(druid:)
-    @version_manifest = VersionsManifest.new(path: @paths.versions_manifest_path)
-    @contents = Contents.new(service: self)
-    @metadata = Metadata.new(service: self)
   end
```

Comment:
Do you think giving line 23 the same treatment is worth considering? (i.e., extracting the ivar into a new instance method that memoizes the same ivar)

Reply:
Oh, nevermind. I now see the other extracted methods depend on `@paths` already.

Reply:
I thought about it, but would rather do that in a separate PR if we think it's a good idea.